src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -1192,6 +1192,81 @@ public function testFileVersionDeleteActiveShould400()
         $this->assertHttpStatusCode(400, $client->getResponse());
     }
 
+    public function testPutRemovingCopyright()
+    {
+        $media = $this->createMedia('image', 'de');
+
+        $fileVersionMetadata = $media->getFiles()[0]->getFileVersions()[0]->getMeta()[0];
+        $fileVersionMetadata->setCopyright('Copyright by Test');
+
+        $this->em->persist($media);
+        $this->em->persist($fileVersionMetadata);
+        $this->em->flush();
+
+        $client = $this->createAuthenticatedClient();
+        $client->request(
+            'PUT',
+            '/api/media/' . $media->getId() . '?locale=de',
+            [
+                'copyright' => null,
+            ]
+        );
+
+        $response = json_decode($client->getResponse()->getContent(), true);
+
+        $this->assertSame(null, $response['copyright']);
+    }
+
+    public function testPutRemovingDescription()
+    {
+        $media = $this->createMedia('image', 'de');
+
+        $fileVersionMetadata = $media->getFiles()[0]->getFileVersions()[0]->getMeta()[0];
+        $fileVersionMetadata->setDescription('Test description');
+
+        $this->em->persist($media);
+        $this->em->persist($fileVersionMetadata);
+        $this->em->flush();
+
+        $client = $this->createAuthenticatedClient();
+        $client->request(
+            'PUT',
+            '/api/media/' . $media->getId() . '?locale=de',
+            [
+                'description' => null,
+            ]
+        );
+
+        $response = json_decode($client->getResponse()->getContent(), true);
+
+        $this->assertSame(null, $response['description']);
+    }
+
+    public function testPutRemovingCredits()
```

Comment:
Would create one test for all fields because functional tests cost performance or what do you think @danrot.